drag_and_drop_v2/utils.py

```diff
@@ -11,4 +11,18 @@ def ngettext(text_singular, text_plural, n):
     if n == 1:
         return text_singular
     else:
-        return text_plural
\ No newline at end of file
+        return text_plural
+
+
+class FeedbackMessages(object):
+    FINAL_ATTEMPT_TPL = _('Final attempt was used, final score is {score}')
```

Comment:
@e-kolpakov These strings will get translated too early, see: http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/internationalization/i18n.html#translating-too-early
One possible way around that would be to change these constants to functions (ie. `FeedbackMessages.final_attempt(score)`, `FeedbackMessages.correctly_placed(correct_count)`, etc).